src/Bonsai.ML.LinearDynamicalSystems/StateComponent.cs

```diff
@@ -61,9 +64,45 @@ public StateComponent(double[,] X, double[,] P, int i)
             Variance = Sigma(P[i,i]);
         }
 
+        /// <summary>
+        /// Creates a new state compenent
+        /// </summary>
+        public StateComponent(double mean, double variance) 
+        {
+            Mean = mean;
+            Variance = variance;
+        }
+
         private double Sigma(double variance)
         {
             return 2 * Math.Sqrt(variance);
         }
+
+        /// <summary>
+        /// Given an observable sequence, this function returns an observable sequence of state components for each element in the input sequence
+        /// /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Given an observable sequence, this function returns an observable sequence of state
        /// components for each element in the input sequence.
        /// </summary>
```

Duplicate `///` and minor formatting